R/plotNhoods.R

```diff
@@ -89,23 +215,24 @@ plotNhoodSizeHist <- function(milo, bins=50){
 #' @rdname plotMiloReducedDim
 #' @import ggplot2
 #' @importFrom dplyr left_join mutate arrange
-#' @importFrom stats filter
 plotMiloReducedDim <- function(x, milo_results, nhood_reduced_dims="UMAP", filter_alpha=NULL, split_by=NULL,
-                               pt_size=1.5, components=c(1,2)){
+                               pt_size=1.5, components=c(1,2)
+){
   ## Check for valid nhoodReducedDim object
   # Should have nrows = no. of nhoods + no. of cells
   if (!nhood_reduced_dims %in% names(nhoodReducedDim(x))){
     stop(paste(nhood_reduced_dims, "is not the name of an embedding in nhoodReducedDims(x). Available reductions are:", paste((nhoodReducedDim(x)), collapse = ", ")))
-  } else if (nrow(nhoodReducedDim(x, nhood_reduced_dims)) != ncol(x) + length(nhoods(x))) {
+  } else if (nrow(nhoodReducedDim(x)[[nhood_reduced_dims]]) != ncol(x) + length(nhoods(x))) {
```

Comment:
Please check how `nhood_reduced_dims` is actually being utilised, i.e. here it is passed into the the function `nhoodReducedim`, rather than being used to index the output.